Makefile

```diff
@@ -1,6 +1,6 @@
 CALICO_DIR=$(shell git rev-parse --show-toplevel)
 VERSIONS_FILE?=$(CALICO_DIR)/_data/versions.yml
-JEKYLL_VERSION=pages
+JEKYLL_VERSION=3.8.3
```

Comment:
Just want to acknowledge that with this change, instead of relying on `jekyll:pages` to notice whenever github has revved their version of Jekyll, we're now relying on ourselves. This should explicitly get acknowledged as a  maintainer reponsibility.

Reply:
Never mind on this. We're switching of github pages so this comment is null & void!